tests/test_integration.py

```diff
@@ -221,8 +247,12 @@ def test_delete(self):
         with self.assertRaises(IGVMError):
             vm_delete(VM_HOSTNAME)
 
-        with _get_vm(VM_HOSTNAME) as vm:
-            vm.shutdown()
+        if self.datacenter_type == 'kvm.dct':
+            with _get_vm(VM_HOSTNAME) as vm:
+                vm.shutdown()
+        elif self.datacenter_type == 'aws.dct':
+            vm_stop(VM_HOSTNAME)
```

Comment:
i am not a fan of this. now we have two different functions from two different places to stop a vm? `vm.shutdown()` should work for both kvm and aws. this `vm.shutdown()` should internally decide what to do based on the `datacenter_type`. we should always aim for a clear api that is consistent

Reply:
I strongly aggree on this one

Reply:
In my opinion, it was wrong in the first place to use `vm.shutdown()`, since we usually call the commands, which we progragate, so I reworked it, so both datacenter_types are using their shared `vm_stop` command.

Reply:
yup i think it's good. it is consistent now with the rest of the code and we do not have business logic in the test :)